models/tasks.js

```diff
@@ -127,8 +128,11 @@ const fetchUserTasks = async (username, statuses = []) => {
         .where('status', 'in', statuses)
         .get()
     } else {
-      tasksSnapshot = await tasksModel.where('participants', 'array-contains', userId)
-        .get()
+      if (hasUser === false) {
+        tasksSnapshot = await tasksModel.where('participants', 'array-contains', userId)
```

Comment:
What is `hasUser`? 🤔

Reply:
`fetchUserTasks` this function is used by multiple function Ex. `getSelfTasks` So added a parameter to that function just to identify the function is being called from API of `task:username`. As the API gives us the user so I added a parameter named `hasUser`.

Reply:
I don't think it's required.  Even for getting self tasks, we can use `fetchUserTasks`. At the end it's job is to find tasks for a particular user. Getting self tasks would be a special case but we did not modify the function. What do you think?

Reply:
What the purpose of identifying from where the function was called? The model function's only is do database calls and return the requested data, it doesn't worry about who's calling those functions and from where. Let me know you think 😉

Reply:
Actually as per the discussion I had with @sumitd94 He said we need the tasks by assignee only. In self tasks it fetches tasks where user is in participant. If we want to include the taskd where user is in participant then we don't need `hasUser` We can move ahead with current implementation.

Reply:
Now I get what you're trying to say
@sumitd94 need some clarifications here. Group tasks would have `participants` list, right? Don't we want those group tasks to show up on a members' page? I'm getting a bit confused here 😅 
Like the `/tasks` page on my-site, @sumitd94 and I made that page, shouldn't it show up on both of our contributions? 🤔 

cc: @ankushdharkar